packages/squiggle-lang/__tests__/Reducer/Reducer_Peggy/Reducer_Peggy_Parse_test.res

```diff
@@ -3,359 +3,464 @@ open Reducer_Peggy_TestHelpers
 
 describe("Peggy parse", () => {
   describe("float", () => {
-    testParse("1.", "{1}")
-    testParse("1.1", "{1.1}")
-    testParse(".1", "{0.1}")
-    testParse("0.1", "{0.1}")
-    testParse("1e1", "{10}")
-    testParse("1e-1", "{0.1}")
-    testParse(".1e1", "{1}")
-    testParse("0.1e1", "{1}")
+    testParse("1.", "{(::$_endOfOuterBlock_$ () 1)}")
```

Comment:
This addition seems pretty annoying, but I assume there's a good reason for it. I imagine adding it everywhere wasn't very fun.

Reply:
Maybe it makes sense to rename ``endOfOuterBlock`` to ``outerBlockEnd``? Just to shorten it a bit, if it will be everywhere.